File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.js
function solution(num_list) {
    let result1 = 0;
    let result2 = 0;
    for(let i = 0; i < num_list.length; i++){
        if (i % 2 == 0) result1 += num_list[i];
        else result2 += num_list[i]
    }
    return result1 > result2 ? result1 : result2
    
}

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    hol = 0
    jack = 0
    for i,j in enumerate(num_list):
        if (i + 1) % 2 == 1:
            hol +=j
        else:
            jack +=j
    return hol if hol > jack else jack